ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -342,6 +372,181 @@ objects:
+        import os
+        import sys
+
+        if len(sys.argv) < 4:
+            print("Usage: ./update_route_53.py <DOMAIN> <RECORD> <IP>")
+            sys.exit(1)
+
+        domain = sys.argv[1]
+        record = sys.argv[2]
+        ip = sys.argv[3]
+        print("record: %s" % record)
+        print("ip: %s" % ip)
+
+        client = boto3.client('route53')
+        r = client.list_hosted_zones_by_name(DNSName=domain, MaxItems="1")
+        zone_id = r['HostedZones'][0]['Id'].split('/')[-1]
+
+        response = client.change_resource_record_sets(
+            HostedZoneId=zone_id,
+            ChangeBatch= {
+                'Comment': 'add %s -> %s' % (record, ip),
+                'Changes': [
+                {
+                    'Action': 'UPSERT',
+                    'ResourceRecordSet': {
+                        'Name': record,
+                        'Type': 'A',
+                        'TTL': 60,
+                        'ResourceRecords': [{'Value': ip}]
+                    }
+                }]
+        })
+        PYTHON_EOF
+          DOMAIN=$(oc whoami --show-server | grep -oP "api.\\K([^\\:]*)")
+          for i in "${!NEW_MASTER_IPS[@]}"; do
+            ETCD_NAME="etcd-${i}.${DOMAIN}"
+            python /tmp/update_route_53.py "${DOMAIN}" "${ETCD_NAME}" "${NEW_MASTER_IPS[$i]}"
+          done
+
+          echo "Run etcd-signer"
+          SURVIVING_MASTER_NODE_SHORT=${SURVIVING_MASTER_NODE%%.*}
+          curl -O https://raw.githubusercontent.com/hexfusion/openshift-recovery/master/manifests/kube-etcd-cert-signer.yaml.template
+          sed "s;__MASTER_HOSTNAME__;${SURVIVING_MASTER_NODE_SHORT};g" kube-etcd-cert-signer.yaml.template > kube-etcd-cert-signer.yaml
+          retry 10 oc create -f kube-etcd-cert-signer.yaml
+          retry 10 oc get pod/etcd-signer -n openshift-config -o name
+          retry 10 oc wait pod/etcd-signer -n openshift-config --for condition=ready
+
+          echo "Grow etcd cluster to full membership"
+          SURVIVING_MASTER_IP=$(oc get nodes ${SURVIVING_MASTER_NODE} -o jsonpath='{.status.addresses[?(@.type=="InternalIP")].address}')
+          SETUP_ETCD_ENVIRONMENT=$(oc adm release info --image-for setup-etcd-environment)
+          KUBE_CLIENT_AGENT=$(oc adm release info --image-for kube-client-agent)
+          MASTERS=($(oc -n openshift-machine-api \
+            get machines \
+            -l machine.openshift.io/cluster-api-machine-role=master \
+            -o jsonpath='{range .items[*]}{.status.addresses[?(@.type=="InternalDNS")].address}{"\n"}{end}'))
+          for master in ${MASTERS[@]}
+          do
+            if [[ "${master}" == ${SURVIVING_MASTER_NODE} ]]; then continue; fi
+            echo "Recovering ${master}"
+            bastion_ssh core@${master} "sudo -i env SETUP_ETCD_ENVIRONMENT=${SETUP_ETCD_ENVIRONMENT} KUBE_CLIENT_AGENT=${KUBE_CLIENT_AGENT} /bin/bash -x  /usr/local/bin/etcd-member-recover.sh ${SURVIVING_MASTER_IP}"
+          done
```

Comment:
I am about to break this and make etcd_name a required param. Can we add this now? For AWS you can do something like
```
      HOSTNAME=$(hostname)
      HOSTDOMAIN=$(hostname -d)
      ETCD_NAME=etcd-member-${HOSTNAME}.${HOSTDOMAIN}
```

The above will faile for vsphere and bare metal so we will need to figure out what works best there.

https://github.com/openshift/machine-config-operator/blob/ca2e5c541f3bd1dc1ec82e4fabafbb4f5228ca63/templates/master/00-master/_base/files/usr-local-bin-etcd-member-recover-sh.yaml#L22

Reply:
>Can we add this now? 

Sure

>The above will faile for vsphere and bare metal so we will need to figure out what works best there.

I don't think we plan to run DR scenarios on vsphere CI, so it just needs to be properly documented